beam-core/Database/Beam/Backend/SQL/BeamExtensions.hs

```diff
@@ -146,6 +146,10 @@ class BeamSqlBackend be => BeamHasInsertOnConflict be where
   type SqlConflictTarget be (table :: (* -> *) -> *) :: *
   type SqlConflictAction be (table :: (* -> *) -> *) :: *
 
+  -- | This is necessary for places that we can't use a forall because the
```

Comment:
You can probably use `QInternal` here?

Reply:
Done. Out of curiosity, what's the theory behind `QInternal`? It doesn't look used anywhere. Should all instances of `PostgresInaccessible` be replaced with that?

Reply:
It's only accessible in beam-core, I think?